product_docs/docs/pem/9/installing/configuring_the_pem_server_on_linux.mdx

```diff
@@ -22,18 +22,24 @@ The PEM server package includes a script (`configure-pem-server.sh`) to help aut
-| `-ci`      | CIDR-formatted network address range that agents connect to the server from, to be added to the server's `pg_hba.conf` file, for example, `192.168.1.0/24`. The default is `0.0.0.0/0`.                                                                            |
-| `-dbi`     | The directory for the database server installation, for example, `/usr/edb/as12` for EDB Postgres Advanced Server or `/usr/pgsql-12` for PostgreSQL.                                                                                                                                 |
-| `-ds`      | The unit file name of the PEM database server. For EDB Postgres Advanced Server, the default file name is `edb-as-12`. For PostgreSQL, it's `postgresql-12`.                                                                                                                        |
-| `-ho`      | The host address of the PEM database server.                                                                                                                                                                                                                            |
-| `-p`       | The port number of the PEM database server.                                                                                                                                                                                                                             |
-| `-ps`      | The service name of the pemagent. The default value is `pemagent`.                                                                                                                                                                                                      |
-| `-sp`      | The superuser password of the PEM database server. This value is required.                                                                                                                                                                                              |
-| `-su`      | The superuser name of the PEM database server.                                                                                                                                                                                                                          |
-| `-t`       | The installation type: Specify `1` if the configuration is for web services and backend database, `2` if you're configuring web services, or `3` if you're configuring the backend database. If you specify `3`, the database must reside on the local host. |
+| Option                                    | Description                                                                                                                                                                                                                                                             |
```

Comment:
I think we can make the descriptions tighter and more parallel in this table. For example, the first starts with a verb but none of the others do. We don't need to start the descriptions with "The ..." or say it is an option when they are all options, etc. 
@ebgitelman